src/ResourceManager/Sql/LegacySdk/SqlManagement.csproj

```diff
@@ -50,5 +50,20 @@
       <Name>Commands.Profile</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
```

Comment:
@ziwa-msft please revert the changes in this file -- you don't need to add a reference to ClientRuntime and I was under the impression since this was a generation of the old Sql SDK that you don't need a reference to the new Sql management SDK.